Barycentric_coordinates_2/examples/Barycentric_coordinates_2/segment_coordinates.cpp

```diff
@@ -1,3 +1,4 @@
+#include <CGAL/Installation/internal/disable_deprecation_warnings_and_errors.h>
```

Comment:
I disagree. We want to test deprecated features, but examples should not use deprecated code.

Reply:
Removed.